### PR TITLE
Navigation login 1

### PR DESCRIPTION
**Pull request by AleMathus**
### **Summary**
Navigation implemented and added to the Home Page and Sign Up screen, added intent to the Wolox page for terms and conditions.

### **Trello**

https://trello.com/c/YChBs3eJ

### **Screenshot**

https://user-images.githubusercontent.com/108141974/180035032-ca70b5e5-891c-4920-8bf7-35be353c6d08.mov

### CODE REVIEW

File: app/src/main/java/com/example/myapplication/ui/LoginFragment.kt [hunk 1]
```diff
@@ -72,4 +90,37 @@ class LoginFragment : Fragment() {
...
+                loginViewModel.signUpNavigated()
+            }
+        }
+    }
+
+    private fun termsAndConditionsClickObserver() {
+        loginViewModel.termsAndConditionsClick.observe(viewLifecycleOwner) {
+            it?.let {
+                val browserIntent =
+                    Intent(Intent.ACTION_VIEW, Uri.parse("https://www.wolox.com.ar/"))
```
Comment (jcruz-wolox) [inline app/src/main/java/com/example/myapplication/ui/LoginFragment.kt]: You could put this url inside a private static constant (companion object).

**Review by jcruz-wolox — CHANGES_REQUESTED** [review]
I only have one comment, the rest is well for me.

**Review by LisandroDiMeo — APPROVED** [review]
Agree with Jona's comment 👍

File: app/src/main/java/com/example/myapplication/ui/LoginFragment.kt [hunk 2]
```diff
@@ -31,18 +39,28 @@ class LoginFragment : Fragment() {
                 val passwordText = lastNameEdit.text.toString()
                 loginViewModel.fieldsValuesValidation(emailText, passwordText)
             }
+            signUpButton.setOnClickListener {
+                loginViewModel.signUpClicked()
+            }
+            termsAndConditions.setOnClickListener {
+                loginViewModel.termsAndConditionsClicked()
+            }
```
Comment (ffasitella) [inline app/src/main/java/com/example/myapplication/ui/LoginFragment.kt]: There's no need to call the viewmodel here. You can just make the redirection like:
```
signUpButton.setOnClickListener {
    goToSignUp()
}
```
As a general rule, don't call the viewmodel unless you are making a request or something like that.

File: app/src/main/java/com/example/myapplication/vm/LoginViewModel.kt [hunk 3]
```diff
@@ -26,6 +25,18 @@ class LoginViewModel(app: Application) : AndroidViewModel(app) {
     val invalidEmail: LiveData<Boolean>
         get() = _invalidEmail
 
+    private val _validEmail = MutableLiveData<Boolean>()
+    val validEmail: LiveData<Boolean>
+        get() = _validEmail
```
Comment (ffasitella) [inline app/src/main/java/com/example/myapplication/vm/LoginViewModel.kt:26]: You are storing a boolean for both valid email and invalid email. You can create a LiveData called validEmail, and set it true or false.

**Review by ffasitella — CHANGES_REQUESTED** [review]
There's a single issue to fix before merging. Good job!

**Review by ffasitella — APPROVED** [review]
🚀

**Review by jcruz-wolox — APPROVED** [review]

**Pull request merged**